### ISSUE TITLE
Too many blank lines being added 

### ISSUE THREAD

**Issue by mohitathwani** [EnlighterJS/Plugin.WordPress#162]
The plug in on my Wordpress web site is messing up the styling. There are too many blank lines being added. This is the same for all EnlighterJS themes.

<img width="720" alt="Screen Shot 2019-03-23 at 3 52 01 PM" src="https://user-images.githubusercontent.com/1278646/54872731-1bddfc00-4d86-11e9-9710-57c431cc38eb.png">

Any ideas how to fix this?

**Comment by AndiDittrich** [comment]
this are no blank lines... your themes css is adding some margin/padding to the lines (`li` elements).

**Comment by mohitathwani** [comment]
Is there any way to fix this?

Cheers !

--
Mohit Athwani
about.me/mohitathwani

**Comment by AndiDittrich** [comment]
of course, you have to modify your themes css (mostly there are some weak `!important` css directives set..). 

to see which rules are affecting the EnlighterJS styles just use a tool like Chrome/Chromium DevTools. you can also post a link to a particular post on your website - i will take a look on it
